web_console/configuring-web-console.adoc

```diff
@@ -13,5 +13,9 @@ or disable the console.
 
 * Deploy an {product-title} cluster.
 
+// cannot create resource "consoles"
```

Comment:
These comments seem quite terse and may be confusing for new writers. It is not clear what you are referring to. Who cannot create console resources?